python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py

```diff
@@ -28,46 +28,58 @@ async def deploy_configs_to_cv(configs: list[CVEosConfig], result: DeployToCvRes
 
     TODO: See if this can be optimized to check if the configlets are already in place and correct. A hash would have been nice.
     """
-    LOGGER.info("deploy_configs_to_cv: %s", len(configs))
+    LOGGER.info("deploy_configs_to_cv: %s devices to check for config", len(workflow_devices))
 
-    if not configs:
+    if not workflow_devices:
         return
 
-    # Build todo with CVEosConfig objects that exist on CloudVision. Add the rest to skipped.
-    result.skipped_configs.extend(config for config in configs if not config.device._exists_on_cv)
-    LOGGER.info("deploy_configs_to_cv: %s skipped configs because the devices are missing from CloudVision.", len(result.skipped_configs))
-    todo_configs = [config for config in configs if config.device._exists_on_cv]
-    LOGGER.info("deploy_configs_to_cv: %s todo configs.", len(todo_configs))
+    # Build todo with WorkflowDevice objects that exist on CloudVision. Add the rest to skipped.
+    todo_devices_with_configs: list[WorkflowDevice] = []
+    skipped_devices = 0
+    for device in workflow_devices:
+        if not device.in_cv_inventory:
+            skipped_devices += 1
+            device.result.config.skipped.extend([device.avd_device.config])
+            LOGGER.info("deploy_configs_to_cv: Skipping config for device %s missing in CloudVision Inventory", device.avd_device.hostname)
+        else:
+            todo_devices_with_configs.append(device)
+
+    LOGGER.info("deploy_configs_to_cv: %s skipped configs because the devices are missing from CloudVision.", skipped_devices)
```

Comment:
Can we just use `len()` here and avoid the counter?
```suggestion
    LOGGER.info("deploy_configs_to_cv: %s skipped configs because the devices are missing from CloudVision.", len(device.result.config.skipped))
```